local_test/run_worker.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -ex
+
+mkdir -m700 -p /root/.ssh
+cp local_test/id_rsa.rrrspec /root/.ssh/id_rsa
```

Comment:
Git preserves only an executable bit. Even if you commit a file in 600, it will be checked out as 644.